packages/ui-components/src/Charts/barChart.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "Style/util/color.scss";
+@import "Style/util/spacing.scss";
+@import "Style/util/styling.scss";
+@import "Style/util/fonts.scss";
+@import "Style/util/type.scss";
+@import "Style/util/depth.scss";
+
+.bar-chart {
+  :global(.recharts-cartesian-axis-tick-value) {
+    fill: black;
+    padding-right: 4px;
+  }
+
+  :global(.recharts-cartesian-axis-line) {
+    stroke: $color-core-neutral-3;
+    stroke-width: 2px;
+  }
+
+  :global(.recharts-cartesian-axis-tick-line) {
+    stroke: $color-core-neutral-3;
+    stroke-width: 2px;
+  }
+
+  :global(.recharts-tooltip-wrapper) {
+    border-radius: 6px;
+    overflow: hidden;
+    border: 0;
+    outline: 0;
+    background-color: $color-core-neutral-0;
+    @include crl-depth(1);
+    @include crl-type-body;
+  }
+
+  :global(.recharts-default-tooltip) {
+    border: 0 !important;
```

Comment:
Is the `!important` necessary here? If so, is there another way around it?

Reply:
Removed this global override. This was necessary to override the `style` attributes applied to the tooltip, but I opted to use a `wrapperStyle` prop on the Tooltip instead.